pkg/resourcegroup/controllers/resourcegroup/resourcegroup_controller_test.go

```diff
@@ -414,68 +464,3 @@ func TestReconcileTimeout(t *testing.T) {
 		})
 	}
 }
-
-func TestActuationStatusToLegacy(t *testing.T) {
```

Comment:
Why was this test removed?

Reply:
Merge mistake. Re-added. I'm gonna delete it in a subsequent PR.